src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  color: 'red' | 'green' | 'blue';
```

Comment:
'red' | 'green' | 'blue' is duplicated, so pleas move it to ColorType
The same with shape